packages/contracts/migrations/6_registry.ts

```diff
@@ -7,30 +7,23 @@ const Token = artifacts.require("EIP20.sol");
 const DLL = artifacts.require("dll/DLL.sol");
 const AttributeStore = artifacts.require("attrstore/AttributeStore.sol");
 
-const OwnedAddressTCRWithAppeals = artifacts.require("tcr/OwnedAddressTCRWithAppeals.sol");
+const CivilTCR = artifacts.require("tcr/CivilTCR.sol");
```

Comment:
while we're at it:
`./tcr/CivilTCR.sol` -> add the relative path to make resolving easier and remove a class of bugs